aws_batch/pipeline.py

```diff
@@ -435,463 +497,530 @@ def run_sample(sample_s3_input_path, sample_s3_output_path,
+                          '-l','10',
+                          '-a','T',
+                          '-b','0',
+                          '-v','1',
+                          '-z', str(multiprocessing.cpu_count()), # threads
+                          '-q', input_path,
+                          '-o', output_path[:-3],
+                          ';'])
+    commands += "aws s3 cp /home/ec2-user/batch-pipeline-workdir/%s/%s %s/;" % \
+                 (sample, RAPSEARCH2_OUT, sample_s3_output_path)
+    check_command = 'ssh -o "StrictHostKeyChecking no" -i %s ec2-user@%s "ps aux|grep rapsearch|grep -v bash"' % (key_path, RAPSEARCH2_INSTANCE_IP)
+    logging.getLogger().info("waiting for server")
+    wait_for_server('RAPSEARCH2', check_command, RAPSEARCH2_MAX_CONCURRENT)
+    logging.getLogger().info("starting alignment")
+    remote_command = 'ssh -o "StrictHostKeyChecking no" -i %s ec2-user@%s "%s"' % (key_path, RAPSEARCH2_INSTANCE_IP, commands)
+    execute_command(remote_command)
+    logging.getLogger().info("finished alignment")
+    # move output back to local
+    time.sleep(10) # wait until the data is synced
+    execute_command("aws s3 cp %s/%s %s/" % (sample_s3_output_path, RAPSEARCH2_OUT, result_dir))
 
 def run_generate_taxid_outputs_from_m8(sample_name,
     annotated_m8, fastq_file_1,
     taxon_counts_csv_file, taxon_counts_json_file,
     taxon_species_rpm_file, taxon_genus_rpm_file,
     taxinfodb_s3_path, count_type, db_sample_id,
-    sample_host, sample_location, sample_date, sample_tissue,
-    sample_template, sample_library, sample_sequencer, sample_notes,
     result_dir, sample_s3_output_path, lazy_run):
-    logger = logging.getLogger()
-    logger.info("========== generate taxid outputs from m8 ==========")
     # Ignore lazyrun
     # download taxoninfodb if not exist
     taxoninfo_filename = os.path.basename(taxinfodb_s3_path)
     taxoninfo_path = REF_DIR + '/' + taxoninfo_filename
     if not os.path.isfile(taxoninfo_path):
         execute_command("aws s3 cp %s %s/" % (taxinfodb_s3_path, REF_DIR))
-        logger.info("downloaded taxon info database")
+        logging.getLogger().info("downloaded taxon info database")
     generate_tax_counts_from_m8(annotated_m8, taxon_counts_csv_file)
-    logger.info("generated taxon counts from m8")
+    logging.getLogger().info("generated taxon counts from m8")
     generate_json_from_taxid_counts(sample_name, fastq_file_1, taxon_counts_csv_file,
                                     taxoninfo_path, taxon_counts_json_file,
-                                    count_type, db_sample_id, sample_host, sample_location,
-                                    sample_date, sample_tissue, sample_template, sample_library,
-                                    sample_sequencer, sample_notes)
-    logger.info("generated JSON file from taxon counts")
+                                    count_type, db_sample_id)
+    logging.getLogger().info("generated JSON file from taxon counts")
```

Comment:
is logging a global variable? if so, you need to uppercase it.

Reply:
no, it's a module